app/views/transitive_dependencies/show.js.erb

```diff
@@ -0,0 +1,32 @@
+var all_deps = $('.deps_item').map(function() {
+	return $(this).attr('data-id');
```

Comment:
I think indent by 2 spaces will fit in this file as well since it is used in all another templates.

Reply:
Yes. I've changed it. Thank you for that.